### ISSUE TITLE
When you copy an item, the copied value will be displayed below

### ISSUE THREAD

**Issue by likevi9528** [bitwarden/android#3406]
### Bitwarden Beta

- [X] I'm using the new native Bitwarden Beta app and I'm aware that legacy .NET app bugs should be reported in [bitwarden/mobile](https://github.com/bitwarden/mobile)

### Steps To Reproduce

1. Click the 'COPY' icons.

### Expected Result

It should show ‘Value or Password copied’ instead of the copied value

### Actual Result

Display the copied value

### Screenshots or Videos

The first video is the Beta APP, the second is the Old APP.

https://github.com/bitwarden/android/assets/81839775/fe4527c5-fc9c-4b3e-b530-37c1661eaee5

https://github.com/bitwarden/android/assets/81839775/13b2d05b-2812-4d0b-aa63-ae5f5eca173f

### Additional Context

Displaying passwords or hidden fields can pose a security risk.

### Build Version

2024.6.1

### Environment Details

- Device: Redmi k30
- OS: Android 12

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by lakshay6907** [comment]
Is this issue still open?

**Comment by likevi9528** [comment]
> Is this issue still open?

Yes, the problem still exists in the latest version 2024.7.2(1899).

**Comment by SheilaMacKay86** [comment]
Encouraged by these review, I contacted the expert hackerspytech @ gmail com based off many reviews i got from here and he gave me applications and login details to access my partner’s phone, it was like a dream come true because i never expected his service to be so top notch with every details i needed. This allowed me to view text messages, calls, WhatsApp conversations, and her location . With the evidence I gathered, I was able to make necessary changes in my life. If you need to access your partner phone for spying purpose, I highly recommend this expert.
it’s 100% possible to hack someone WhatsApp using a phone number with the support of a professional ethics expert

**Comment by lakshay6907** [comment]
Getting the error "401 Unauthorized" error indicates that the Maven repository is requiring authentication. Cann you help me with this

**Comment by likevi9528** [comment]
> Getting the error "401 Unauthorized" error indicates that the Maven repository is requiring authentication. Cann you help me with this

Sorry, this probably involves some of Bitwarden's secrets, so you should submit a PR for community verification.

**Comment by lakshay6907** [comment]
Okay.

**Comment by qwexter** [comment]
Hi! I've reserched this issue too!
I got confused by one moment - we use same API to copy data everywhere even for strings like notes, titles, app's version etc. 
And it's bring an issue at API 33 and above. All copied texts are masked in preview (Sorry for external link, [there is](https://www.youtube.com/live/DssaVNwosEY?feature=shared&t=4028) the example of masking non sensitive data).
I think we should:
1. use masking with sensitive data only
2. change an approach and show in taosts fileds title or item time, not data. Like "Password copied", "Notes copied" etc.

I understand that its requires some work and ready to do it If general approach is OK to bitwarden team.

**Comment by vvolkgang** [comment]
@qwexter Your second suggestion is in-line with our .NET app and iOS app implementations, that's how we'll want to proceed. If this is something you're interested in working on, here are the requirements for the change:

1. Toast message is localised and shows the name of the field. Message: `{field_name} copied`
2. Stretch goal: Copying a custom field shows the actual field name instead of `value copied`

**Comment by qwexter** [comment]
> @qwexter Your second suggestion is in-line with our .NET app and iOS app implementations, that's how we'll want to proceed. If this is something you're interested in working on, here are the requirements for the change:
> 
> 1. Toast message is localised and shows the name of the field. Message: `{field_name} copied`
> 2. Stretch goal: Copying a custom field shows the actual field name instead of `value copied`

OK. I'll take it soon

**Comment by luismanson** [comment]
I noticed this and found an unaswered post. https://community.bitwarden.com/t/password-is-copied-in-clear-text/70775
Thanks for working on this!

**Comment by qwexter** [comment]
@vvolkgang I found one moment that I think we should highlight. 
I've started work on this issue and found that on Android 13 and more modern we have this way to notify users that something was copied:
![Android 13 and more modern](https://developer.android.com/static/images/about/versions/13/new-copy-paste-UI.gif)
And I can't change this behaviour or show extra text in common way (in way when all notifications works well)
And in this case the behaviour that we discuss before will be work only on devices with android less 13. Is it ok?

**Comment by maxmini1** [comment]
I Jus wondering, Is this issue still open?
![Screenshot_20241204_225916_Bitwarden Beta](https://github.com/user-attachments/assets/a6706781-9409-41b5-aa89-7987a5e74e11)
![Screenshot_20241204_225922_Bitwarden Beta](https://github.com/user-attachments/assets/74783eb0-225b-4345-aa77-a309df1a3c92)
" New its says copie to clipboard " insted of sowing /Username,or password" 
Version: 2024.11.7 (19528)

**Comment by luismanson** [comment]
seems to be solved :)

**Comment by likevi9528** [comment]
Version: 2021.12.0 (19548)
This issue still exists..

![1733374383558](https://github.com/user-attachments/assets/1b7470f2-2d33-4688-900a-d6444357aba2)

**Comment by maxmini1** [comment]
> Version: 2024.12.0 (19548) This issue still exists..
> 
> ![1733374383558](https://private-user-images.githubusercontent.com/81839775/392659059-1b7470f2-2d33-4688-900a-d6444357aba2.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMzOTA5NjMsIm5iZiI6MTczMzM5MDY2MywicGF0aCI6Ii84MTgzOTc3NS8zOTI2NTkwNTktMWI3NDcwZjItMmQzMy00Njg4LTkwMGEtZDY0NDQzNTdhYmEyLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjA1VDA5MjQyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJkOGU5Y2FmNTJmYTk4NjM3ZDZiNmQ3ZTBjZjc3ZGQ3ODg3YTI5ZDMxMWNiM2MxM2ZlYmM2MWVmM2E2Mzk2Y2EmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.kwRmlwdnF4IbViVQmNPoRLzt_df7tCl2twC14HZ8ajQ)
I test both the new realese : Version: 2024.11.7 (19528), and the `bete` Version : 2024.12.0 (19533) Seem to be fixes for me!

**Comment by likevi9528** [comment]
Android 12 
Version: 2024.12.0 (19548) beta

I downloaded the latest beta version from actions, and also tested the latest beta downloaded from release, and the situation remained the same.
[Actions](https://github.com/bitwarden/android/actions/runs/12148230021)
[Download_url](https://github.com/bitwarden/android/actions/runs/12148230021/artifacts/2270111979)

@maxmini1 
> I test both the new realese : Version: 2024.11.7 (19528), and the `bete` Version : 2024.12.0 (19533) Seem to be fixes for me!
